packages/react-devtools-shared/src/backend/fiber/DevToolsFiberComponentStack.js

```diff
@@ -135,6 +135,7 @@ export function supportsConsoleTasks(fiber: Fiber): boolean {
   return !!fiber._debugTask;
 }
 
+// TODO: clean this up?
```

Comment:
@sebmarkbage is it safe for me to delete this? Not sure about devtools versioning

Reply:
No it's not safe to delete because this is the feature detection whether this version of React supports owner stacks or not.